source/sass/buyers-guide/views/catalog.scss

```diff
@@ -118,7 +123,8 @@ body.catalog {
       flex: 0 0 auto;
       display: inline-block;
       background: url(../_images/buyers-guide/filter/search.svg);
-      background-size: 100% 100%;
+      background-size: 22px 22px;
+      background-repeat: no-repeat;
```

Comment:
100% is cutting off the icon by a single pixel, so this is the simplest solution I can see.